python/paddle/v2/framework/layers.py

```diff
@@ -256,6 +311,35 @@ def conv2d(input,
     return helper.append_activation(pre_act)
 
 
+def pool1d(input,
+           pool_size,
+           pool_type,
+           pool_stride=1,
+           pool_padding=0,
+           global_pooling=False,
+           program=None,
+           init_program=None):
+    # FIXME(dzh) : want to unify the argument of python layer
+    # function. So we ignore some unecessary attributes
+
+    ENUM_POOL_TYPE = ["max", "avg", "sqrt", "last", "first"]
```

Comment:
Initialize `ENUM_POOL_TYPE` as a set to make the following `if pool_type not in ENUM_POOL_TYPE:` faster.

Reply:
Done.